apps/studio/src/server/modules/site/site.types.ts

```diff
@@ -1,11 +1,11 @@
 export interface Sitemap {
   parentTitle: string
   childrenTitles: string[]
-  sibilngTitles: string[]
+  siblingTitles: string[]
 }
-
+type SiteTheme = 'isomer-classic' | 'isomer-next'
```

Comment:
nit: obj this so that in code can just call `SiteTheme.classic`

Reply:
sure